<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Raphaelaggio.github.io by Raphaelaggio</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Raphaelaggio.github.io</h1>
        <p><a href="#home">[Home]</a></p>
        <p><a href="#packages">[R packages]</a></p>
        <p><a href="#publications">[Scientific production]</a></p>


        <p class="view"><a href="https://github.com/Raphaelaggio">View My GitHub Profile</a></p>

      </header>
      <section>
        <h1>

<a id="home">Welcome</a></h1>

<p align="justify">My name is Raphael Aggio and I would like to welcome you to my website. I am an <em>electronic technician</em> and a <em>biologist</em>. In the beginning of my career I worked for about 5 years at Philips Telecommunications - Brazil, performing installation and maintenance of telephone systems. I then decided to study biology and during most of my bachelors I worked with macro-ecology, etno-biology, fisheries and conservation. When I was about to finish my bachelors I then received a proposal to do a PhD at The University of Auckland - New Zealand, where I was first introduced to Metabolomics and Systems Biology. My PhD was then developed on the analysis of microbial metabolome, construction of metabolic networks for flux balance analysis (FBA) and the development of computer tools to improve the quality and the speed of pipelines for Metabolomics and Systems Biology. I recently created this website in order to store and share some of the computer tools I have developed in the last few years. Feel free to brownse around and please contact me if you have any feedback.</p>
<p align="justify">At the moment I am a Research Associate at The University of Liverpool, where I am applying bioinformatics to classify metabolomics data based on extracted patterns of Volatile Organic Compounds (VOCs). I am certainly opened for collaborations involving the analysis of interesting datasets and/or to develop new computer tools. Of course, you are more than welcome to contribute and improve any of the tools I have developed.</p>
<p>Best wishes.</p>
<p>Raphael Aggio - raphael.aggio [at] gmail.com</p>

<h1><a id="packages">R packages</a></h1>

<h2>Metab</h2>
<p align="justify">Metab is an R package developed to process metabolomics data in a high-throughput manner. Metab processes data previously analysed by the <a href="http://chemdata.nist.gov/dokuwiki/doku.php?id=chemdata:amdis">Automated Mass Spectral Deconvolution and Identification System (AMDIS)</a> and allows users to perform data normalisation and statistical analyses. Metab is available at the <a href="http://www.bioconductor.org/">Bioconductor database</a> and it can also be downloaded <a href="r_packages /metab/Metab_0.99.7.tar.gz">here</a>.</p>

<h3>Requirements</h3>
<p>Metab requires 3 additional packages: <a href="http://www.bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a>, <a href="http://cran.r-project.org/web/packages/svDialogs/index.html">svDialogs</a> and <a href="http://cran.fhcrc.org/web/packages/pander/index.html">pander</a>. These packages can also be downloaded from <a href="http://www.bioconductor.org/">Bioconductor</a> or <a href="http://cran.r-project.org/index.html">CRAN</a>.</p>

<h3>Installing Metab from Bioconductor</h3>
<p>If you decide to install Metab from Bioconductor, you can copy the following code and paste it in your R console. It should download Metab, its required packages and perform all the installation:</p>
<pre><code>
    ### Load biocLite - a Bioconductor script to automate the download and installation of required packages #####
    source("http://bioconductor.org/biocLite.R")
    ### Install Metab ####
    biocLite("Metab")
</code></pre>


<h3>Installing Metab from source file</h3>
<p>If you decide to install Metab using the source file available <a href="r_packages /metab/Metab_0.99.7.tar.gz">here</a>, you can follow the steps described bellow:</p>
<ul>
    <li>Place the source file you just downloaded in your working directory;</li>
    <li>Copy the code below and paste it at your R console:</li>
</ul>
<pre><code>### Install required packages
            source("http://bioconductor.org/biocLite.R")
            biocLite(c("xcms", "svDialogs", "pander”))
            ### Install Metab
            install.packages(“Metab_0.99.7.tar.gz", type = "source")
    </code></pre>


<h2>PAPi</h2>
<p align="justify">PAPi is an R package developed to predict the activity of metabolic pathways based solely on metabolomics data sets. Based on the number and abundances of metabolites identified per metabolic pathway, PAPi calculates the activity score of metabolic pathways in the Kyoto Encyclopedia of Genes and Genomes (KEGG). PAPi is available at <a href="http://www.bioconductor.org/packages/release/bioc/html/PAPi.html">Bioconductor</a></p>
<p align="justify">Alternativelly, you can install PAPi direct from your R console. Just copy the code bellow and paste it at your R console:</p> 
<pre><code>source("http://bioconductor.org/biocLite.R")
biocLite("PAPi")
</code></pre>


<h2>MetaBox</h2>
<p align="justify">MetaBox is an R package developed to identify compounds analysed by GC-MS by scoring chromatographic peaks according to their likelihood of representing metabolites defined in a mass spectral library. MetaBox has been accepted for publication and soon it will be part of the Bioconductor database. In the meanwhile, you can download MetaBox <a href="r_packages/metabox/MetaBox_lastVersion.tar.gz">here</a></p>
<p>For installing MetaBox in R, place the source file you just downloaded in your working directory, copy the code below and paste it at your R console:</p>
<pre><code>### Install required packages
source("http://bioconductor.org/biocLite.R")
biocLite(c("xcms", "svDialogs", "pander", "MassSpecWavelet"))
### Install MetaBox
install.packages(“MetaBox_lastVersion.tar.gz", type = "source")
</code></pre>

<h2><a id="publications">Scientific production</a></h2>

<p align="justify"><strong>Aggio, R.</strong>; Mayo, A.; Reade, S.; Probert, C.S. and Ruggiero, K. (2014). <strong>Identifying and quantifying metabolites by scoring peaks of GC-MS data. <i>BMC Bioinformatics</i></strong>, 15:374, doi:10.1186/s12859-014-0374-2.</p>
<p align="justify">Reade, S.; Mayo, A.; <strong>Aggio, R.</strong>; Khalid, T.; Pritchard, D.M.; Ewer, A.K. and Probert, C.S. (2014). <strong>Optimisation of Sample Preparation for Direct SPME-GC-MS Analysis of Murine and Human Faecal Volatile Organic Compounds for Metabolomic Studies. <i>J Anal Bioanal Tech</i></strong>, 5: 184. doi:10.4172/2155-9872.1000184.</p>
<p align="justify"><strong>Aggio, R.</strong> and Probert, C.S. (2014). <strong>Future Methods for the Diagnosis of Inflammatory Bowel Disease. <i>Digestive Diseases.</i></strong> 32, 463-467.</p>
<p align="justify"><strong>R. B. M. Aggio</strong> (2014). <strong>Pathway Activity Profiling – PAPi: a tool for metabolic pathway analysis. <i>Yeast Metabolic Engineering: Methods and Protocols.</i></strong>. V. Mapelli, Ed., vol. 1152, pp. 233-250.</p>
<p align="justify">Reade, S.; Duckworth, C.A.; Khalid, T.; Mayor, A.; <strong>Aggio, R.</strong>; Pritchard, D.M. and Probert, C.S. (2014). <strong>PWE-109 A Metabolomic profiling study of a chemically-induced mouse model of intestinal inflammation. <i>Gut</i></strong>. 06/2014; 63(Suppl 1):A172. DOI: 10.1136/gutjnl-2014-307263.369.</p>
<p align="justify">Mayor, A.; Reade, S.; <strong>Aggio, R.</strong>; Khalid, T. and Probert, C.S. (2014). <strong>PTH-023 Pediatric faecal VOC analysis: method optimisation. <i>Gut</i></strong>. 06/2014; 63(Suppl 1):A218. DOI: 10.1136/gutjnl-2014-307263.469.</p>
<p align="justify"><strong>Aggio, R.</strong> (2013). <strong>Bioinformatics tools for cell modeling and metabolomics using <i>Lactococcus lactis</i> as a model organism. <i>PhD thesis, The University of Auckland, New Zealand.</i></strong></p>
<p align="justify">M.P.N., Santos; S. Seixas; <strong>R.B.M. Aggio</strong>; N. Hanazaki; M. Costa; A. Schiavetti; J.A. Dias and U.M. Azeiteiro (2012). <strong>A Pesca enquanto Atividade Humana: Pesca Artesanal e Sustentabilidade. <i>Revista de Gestão Costeira Integrada.</i></strong> 12(4):405-427.</p>
<p align="justify"><strong>R. B. M. Aggio</strong> (2012). <strong>Raphael Bastos Mareschi Aggio: Bioanalysis Young Investigator. <i>Bioanalysis</i></strong>, 4, 1285-1286.</p>
<p align="justify">Charlotte M. Wilson; <strong>Raphael B.M. Aggio</strong>; Paul W. O'Toole; Silas G. Villas-Bôas and Gerald W. Tannock (2011). <strong>Transcriptional and metabolomic consequences of luxS inactivation reveal a metabolic rather than quorum sensing role for LuxS in <i>Lactobacillus reuteri</i> 100-23. <i>Journal of Bacteriology</i></strong>. doi:10.1128/ ​JB.06318-11.</p>
<p align="justify">Rebekah A. Frampton; <strong>Raphael B.M. Aggio</strong>; Silas G. Villas-Bôas; Vickery L. Arcus and Gregory M. Cook (2011). <strong>Toxin-antitoxin systems of mycobacterium smegmatis are essential for cell survival. <i>The Journal of Biological Chemistry</i></strong>. doi:10.1074/jbc.M111.286856.</p>
<p align="justify"><strong>Aggio, R.</strong>; Obolonkin, V. and Villas-Bôas, S.G. (2011). <strong>Sonic vibration affects the metabolism of yeast cells growing in liquid culture: a metabolomic study. <i>Metabolomics</i></strong>. doi:10.1007/s11306-011-0360-x.</p>
<p align="justify"><strong>Aggio, R.</strong>; Villas-Bôas, S. G. and Ruggiero, K. (2011). <strong>Metab: an R package for high-throughput analysis of metabolomics data generated by GC-MS. <i>Bioinformatics</i></strong>. doi:10.1093/bioinformatics/btq379.</p>
<p align="justify">Duportet, X.; <strong>Aggio, R.</strong>; Carneiro, S. and Villas-Bôas, S.G. (2011). <strong>The biological interpretation of metabolomic data can be misled by the extraction method used. Metabolomics.</i></strong> doi: 10.1007/s11306-011-0324-1.</p>
<p align="justify"><strong>Aggio, R.B.M.</strong>; Ruggiero, K.; Turner, S.; Nielsen, J. and Villas-Boas, S.G. (2011). <strong>A computer-made bacterium to produce good lipids in cheese. <i>The University of Auckland Expose Competition, New Zealand. (http://www.metabolomics.auckland.ac.nz/images/stories/raphael_Aggio/poster.png)</i></strong></p>
<p align="justify">Smart, K., <strong>Aggio, R.</strong>, Van Houtte, J. and Villas-Bôas, S. (2010). <strong>Analytical platform for metabolome analysis of microbial cells using methyl chloroformate derivatization followed by gas chromatography - mass spectrometry. <i>Nature protocols.</i></strong> doi:10.1038/nprot.2010.108.</p>
<p align="justify"><strong>Aggio, R.</strong>, Ruggiero, K. and Villas-Bôas, S. (2010). <strong>Pathway Activity Profiling (PAPi): from metabolite profile to the metabolic pathway activity. <i>Bioinformatics.</i></strong> doi:10.1093/bioinformatics/btq567.</p>
<p align="justify"><strong>Aggio, R.B.M.</strong> and Hanazaki, N. (2008). <strong>Artisanal Fisheries at North Bay of Florianopolis: Captures, Fish Effort, Problematic and Possible Solutions. <i>Santa Catarina Federal University, Florianopolis, Santa Catarina, Brazil.</i></strong></p>

<h2>Awards</h2>
<p align="justify">Highly commended award (2012). <strong>Aggio, R.B.M.</strong>. Young Investigator Award from Bioanalysiszone.</p>
<p align="justify">High distinction award (2011). <strong>Aggio, R.B.M.</strong>; Ruggiero, K.; Turner, S.; Nielsen, J. and Villas-Boas, S.G. A computer-made bacterium to produce good lipids in cheese. The University of Auckland Expose Competition, New Zealand. (http://www.metabolomics.auckland.ac.nz/images/stories/raphael_Aggio/poster.png).</p>


      </section>
      <footer>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>
